lib/screens/AudioPlayerScreen.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { View, Text, Button } from 'react-native';
+import { Audio } from 'expo-av';
+
+const AudioPlayerScreen: React.FC<{ route: any, navigation: any }> = ({ route, navigation }) => {
```

Comment:
Seems like this and the video player are new components of this story? If working well done and good job separating it in a new file.